src/courseware/course/sidebar/sidebars/course-outline/CourseOutlineTrigger.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import SidebarTriggerBase from '../../common/TriggerBase';
+import { LAYOUT_LEFT } from '../../common/constants';
+import CourseOutlineIcon from './CourseOutlineIcon';
+import messages from './messages';
+
+export const ID = 'COURSE_OUTLINE';
+export const LAYOUT = LAYOUT_LEFT;
```

Comment:
[question]: Do we need these constants?

Reply:
Yes, it uses for separating sidebars by the render side.